scripts/github_pr/github_pr_cloud.yaml

```diff
@@ -11,6 +11,12 @@ template:
     stable: &mkcloud_automation_stable
       <<: *mkcloud_automation_standard
       cloudsource: GM7+up
+  ardana_parameters: &ardana_automation_parameters
+    standard: &ardana_automation_standard
+      <<: *mkcloud_automation_standard
```

Comment:
This would inherit parameters from above `&mkcloud_automation_standard` like mkcloudtarget, nodenumber, networkingplugin that do not exist in the ardana job.